### ISSUE TITLE
Compatible with qemu 6.x

### ISSUE THREAD

**Issue by ssikiki** [digitalocean/go-qemu#187]
virsh qemu-monitor-command DOMAIN --pretty '{ "execute": "query-block" }'

In Ubuntu20, QEMU emulator version 4.2.1 (Debian 1:4.2-3ubuntu6.21)
"dirty-bitmaps" and "inserted" are at the same level of BlockInfo, match qmp/raw/autogen.go BlockInfo

But in Ubuntu22, QEMU emulator version 6.2.0 (Debian 1:6.2+dfsg-2ubuntu6.3)
"dirty-bitmaps" is a key in "inserted", so BlockInfo struct mismatch

It looks like this module needs some changes to adapt to the new version of qemu?

**Comment by connorkuehl** [comment]
Yeah, go-qemu hasn't been regenerated from a newer release of QEMU for some time.

I've filed #188 to describe what's blocking that.
